packages/expo-media-library/android/src/test/java/expo/modules/medialibrary/MockData.kt

```diff
@@ -0,0 +1,72 @@
+package expo.modules.medialibrary
+
+import android.provider.MediaStore.Files.FileColumns.MEDIA_TYPE_AUDIO
+import android.provider.MediaStore.Files.FileColumns.MEDIA_TYPE_IMAGE
+import android.provider.MediaStore.Files.FileColumns.MEDIA_TYPE_VIDEO
+
+/*
+ Column order:
+ ID, DisplayName, uri, MediaType, Width, Height, DateTaken, DateModified, Orientation, Duration, BucketId
+ */
+internal data class MockAsset(
+  val id: Long,
+  val name: String?,
+  val path: String?,
+  val mediaType: Int?,
+  val width: Int?,
+  val height: Int?,
+  val createdDate: Long?,
+  val modifiedDate: Long?,
+  val orientation: Int?,
+  val duration: Int?,
+  val albumId: String?
+) {
+  fun toColumnArray() = arrayOf(
+    id, name, path, mediaType,
+    width, height,
+    createdDate, modifiedDate,
+    orientation, duration,
+    albumId
+  )
+}
+
+internal object MockData {
+  val mockImage = MockAsset(
+    id = 1L,
+    name = "img1.jpg",
+    path = "images/img1.jpg",
+    mediaType = MEDIA_TYPE_IMAGE,
+    width = 100, height = 200,
```

Comment:
```suggestion
    width = 100, 
    height = 200,
```
I know that if you put it in one line, it looks good 😅  But I don't like it 😜